dashboard/test/helpers/script_levels_helper_test.rb

```diff
@@ -60,7 +60,6 @@ class ScriptLevelsHelperTest < ActionView::TestCase
     stubs(:current_user).returns(@student)
     script = @section.script
     script_level = script.get_script_level_by_relative_position_and_puzzle_position 2, 9, false
-    assert script_level.end_of_stage?, 'bad script_level selected for test'
```

Comment:
End of stage does not exclude bonus levels and we now have bonus levels as part of fixture data so this check fails.   Decided to remove since the test still checks the right thing without it and we check that end_of_stage is doing the right thing in other places as well as being tested in other places.